system-int-test/test/cmr/system_int_test/header_test.clj

```diff
@@ -67,3 +69,41 @@
                               {:headers {:cmr-request-id cmr-request-id}
                                :throw-exceptions false})]
       (is (cmr-request-id-in-header? headers cmr-request-id)))))
+
+(defn- get-headers-as-set
+  "Returns comma separated headers as a set"
+  [response header-to-compare]
+  (-> (get-in response [:headers header-to-compare]) 
+      (clojure.string/split #",")
+      (set)))
+
+(defn- compare-cors-headers
+  "Test header received from request vs. header expected"
+  [request-response header-to-compare expected-response]
+  (let [received-headers (get-headers-as-set
+                          request-response
+                          header-to-compare)
+        expected-headers (get-headers-as-set
+                          expected-response
+                          header-to-compare)]
+    (= expected-headers received-headers)))
+
+(deftest cors-headers
```

Comment:
It would be clearer if we just compare the response header value of CORS_CUSTOM_ALLOWED_HEADER with a list of Echo-Token, Client-Id, CMR-Request-Id, CMR-Scroll-Id. The same for the CORS_CUSTOM_EXPOSED_HEADER. We can remove the `compare-cors-headers` function.